packages/framework/presence/src/latestValueControls.ts

```diff
@@ -27,3 +27,27 @@ export interface LatestValueControls {
 	 */
 	forcedRefreshInterval: number;
 }
+
+/**
+ * @internal
+ */
+export class LatestValueControl implements LatestValueControls {
+	public allowableUpdateLatency: number;
+	private _forcedRefreshInterval: number;
+
+	public constructor(settings: LatestValueControls) {
+		this.allowableUpdateLatency = settings.allowableUpdateLatency;
+		this._forcedRefreshInterval = settings.forcedRefreshInterval;
+	}
+
+	public get forcedRefreshInterval(): number {
+		return this._forcedRefreshInterval;
+	}
+	public set forcedRefreshInterval(value: number) {
+		this._forcedRefreshInterval = value < 10 ? 0 : value;
```

Comment:
This seems unexpected - why clamp here?  Also IMO it's always nice to include units in the name of time units if not taking a more-structured object (e.g. forcedRefreshIntervalMs)

Reply:
I didn't see us using units in names publicly - been a long time since I looked.
Intellisense should tell you, but not in PRs likely:
```typescript
	/**
	 * Target time in milliseconds between oldest changed local state
	 * has been broadcast and forced rebroadcast of all local values.
	 * A value of less than 10 disables forced refresh.
	 *
	 * @defaultValue 0
	 */
	forcedRefreshInterval: number;
```
Less than 10ms is just going to be too frequent. Too many signals. Of course, will revisit as we go. Hopefully this option is just removed. :)